index.html

```diff
@@ -1,10 +1,183 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8">
-    <title></title>
-  </head>
-  <body>
 
-  </body>
+<head>
+  <meta charset="utf-8">
+  <title>Surf and Paddle</title>
+  <link href="https://fonts.googleapis.com/css?family=Raleway" rel="stylesheet">
+  <link rel="stylesheet" href="app.css">
+
+</head>
+
+<body>
+  <section class="hero">
+    <nav class="hero__nav">
+      <div class="hero-container">
+        <div class="top-nav">
+          <h1 class="top-nav__logo">
+              <a href="#">
+                <img class="top-nav-img" src="img/board.jpeg"  alt="Surf & Paddle Co." />
+                  Surf & Paddle Co. Blog
+              </a>
+            </h1>
+
+          <ul class="top-nav__links">
+            <li class="top-nav__item"><a href="#">About</a></li>
+            <li class="top-nav__item"><a href="#">Search</a></li>
+          </ul>
+        </div>
+      </div>
+    </nav>
+    <div class="hero-big__text">
+      Surf & Paddle Co.
+    </div>
+    <div class="hero-small__text">
+      An IronYard Academy Project Est. Fall 2013
+    </div>
+  </section>
+
+
+  <section class="main">
+    <div class="hero-container">
+      <div class="body-left">
+        <h1 class="headline">
+            Exciting Things<br>
+            are Happening!!!
+          </h1>
+        <div class="post-author">
+          <div class="img-parent">
+            <img class="img" src="img/anne_murray.jpeg" alt="Anne Marie" />
```

Comment:
To help make this easier to style, use a square image so that the image doesn't get stretched when it resizes and is rounded.